.github/workflows/nightly.yml

```diff
@@ -0,0 +1,30 @@
+name: Nightly v2
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        go-version: [ '1.16', '1.17' ]
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+    timeout-minutes: 120
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: 'v2-devel' # checkout v2-devel branch, scheduled actions only run on the default branch
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: test
+        uses: n8maninger/action-golang-test@v1
+        with:
+          args: "-race;-count=1000;-timeout=30m"
```

Comment:
`100` has been sufficient for me, especially if we're enabling `race`. I would also add `-failfast`